### PR TITLE
Dutch additions to taxonomies

### PR DESCRIPTION
**Pull request by aleene**
**Description:**
Updating the taxonomies based on newly added dutch products.

### CODE REVIEW

File: t/ingredients.t [hunk 1]
```diff
@@ -354,15 +354,6 @@ my @tests = (
 		}
 	],
 
-	# from vegetal origin
```
Comment (stephanegigandet) [inline t/ingredients.t:357]: Removed by error?

**Review by stephanegigandet — COMMENTED** [review]
There's a test that has been removed:

	[
		"en-vegetal-ingredients",
		{
			lc => "en",
			ingredients_text => "Gelatin (vegetal), Charcoal (not from animals), ferments (from plants), non-animal rennet, flavours (derived from plants)",
		}
	],	

Can you add it back?

File: t/ingredients.t (hunk 1 above)
Comment (aleene) [reply t/ingredients.t:357]: I can not see when it was deleted in the history. I saw that you made the change. Maybe I forgot to pull the newest version and did not see any warning.

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**